addons/website/migrations/12.0.1.0/post-migration.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_website_company_id(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE website w
+        SET company_id = c.id
+        FROM (SELECT id FROM res_company ORDER BY id ASC LIMIT 1) AS c
+        WHERE w.company_id IS NULL
+        """
+    )
+
+
+def fill_website_socials(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE website w
+        SET social_facebook = c.social_facebook,
+            social_github = c.social_github,
+            social_googleplus = c.social_googleplus,
+            social_linkedin = c.social_linkedin,
+            social_twitter = c.social_twitter,
+            social_youtube = c.social_youtube
+        FROM res_company c
+        WHERE w.company_id = c.id
+        """
+    )
+
+
+def fill_website_name(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE website w
+        SET name = 'Home #' || w.id
+        WHERE w.name IS NULL
+        """
+    )
+
+
+def fill_website_redirect_type(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE website_redirect wr
+        SET type = '301'
+        WHERE wr.type IS NULL
+        """
+    )
+
+
+def fill_website_redirect_urls(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE website_redirect wr
+        SET active = FALSE
+        WHERE wr.url_from IS NULL OR wr.url_to IS NULL
+        """
+    )
+
+
+def apply_bootstrap_4(view):
+    text = view.arch_db
+    # TO BE FILLED
```

Comment:
We can complete this in a different PR, no unlock dependencies to this module and allow parallel work?

Reply:
Yes, of course!

Reply:
Yes, that would be nice. I would like to work on this `apply_bootstrap_4` method in a different PR.

Reply:
@Yajo is going to work on that topic next week. Do you have already something on that? I will review remaining PRs also next week.

Reply:
Ok. Then move forward, do merge and I will open a new PR for this method. I have an initial draft for that script. Then @Yajo can complete it.